src/core/Hotkey.ts

```diff
@@ -79,6 +79,11 @@ keymaster.filter = function(event) {
   return true;
 };
 
+const ALIASES = {
+  'plus': '=',
```

Comment:
plus is '='? is that intentional? seems a little misleading

Reply:
It's the same button. And there is no way we can make it real "+" using `keymaster` library.